### DISCUSSION TITLE
Bookmark filter settings

### DISCUSSION THREAD

**Discussion by AlexVanK** [directus/directus#10014] [Archive]
It would be nice to be able to bookmark a particular filter setting

![image](https://user-images.githubusercontent.com/56647386/143089748-24924250-7464-4dfb-b0bf-927571f15a49.png)

In this collection I will have hundreds of records, these records can be filtered by status.
It would be nice to be able to access that filtered state via a quick link in the left navigation bar... perhaps as a bookmark?

I was able to achieve this on Directus 8 with custom JS
![image](https://user-images.githubusercontent.com/56647386/143089980-f318bd17-1b72-446e-b8f8-2fc11b96ad42.png)

**Comment by rijkvanzanten** [comment]
That's exactly what bookmarks do 🙂 

Just hit the "Save Changes" button (the yellow checkmark-in-bookmark icon) and the filter will be saved to the current bookmark 👍🏻

**Reply by AlexVanK** [reply]
Yeah that makes sense @rijkvanzanten ...
I noticed that if you change the column display in the parent collection,
that won't be reflected in the bookmarks...

Not sure if that's the intended functionality,
but if you have 20 bookmarks, and you change how many columns you display in the parent collection, you would have to remake all 20 bookmarks...

**Reply by rijkvanzanten** [reply]
That's as intended. A bookmark snapshots the current view for later use, this includes the fields shown
